src/main/python/ayab/transforms.py

```diff
@@ -70,7 +76,8 @@ def repeat(image, args):
                 new_im.paste(image, (w, h))
         return new_im
 
-    def reflect(image, args):
+    @staticmethod
+    def reflect(image:Image.Image, args:tuple[tuple[int,int,int,int]])->Image.Image:
```

Comment:
I kinda don't like this nested tuple, but it was like that when I got here.

Reply:
a tuple of 4 booleans would make more sense